scripts/coverage/gcov-gen-html

```diff
@@ -82,7 +82,8 @@ then
 		cat << EOF
 		push(@gcov_options, "-a") if ($gcov_caps->{'all-blocks'});
 EOF
-		exit 1
+		# Commented below line to avoid change in gcov utility
```

Comment:
Consider deleting this line instead of commenting it if this line is not needed.

Reply:
done